### ISSUE TITLE
Propagate profiling for 12_1_0_pre5 to the web page

### ISSUE THREAD

**Issue by jpata** [jpata/cms-reco-profiling-web#30]
12_1_0_pre5 profiling results are available. Note that 11834.21/step6* should be ignored, it looks like it's a bug in CMSSW that should not affect the results.

```
$ ls /eos/cms/store/user/cmsbuild/profiling/data/CMSSW_12_1_0_pre5/slc7_amd64_gcc900/*
/eos/cms/store/user/cmsbuild/profiling/data/CMSSW_12_1_0_pre5/slc7_amd64_gcc900/11834.21:
cmdLog_profiling.sh  step3_circles.json          step3_igprofMEM.1.gz    step3_TimeMemoryInfo.log    step4_igprof_cpu.txt    step4.root.unused           step5_igprofCPU.gz      step5.log
step1_JobReport.xml  step3_FastTimerService.log  step3_igprofMEM.200.gz  step4_circles.json          step4_igprofMEM.1.gz    step4_TimeMemoryInfo.log    step5_igprof_cpu.txt    step5.root.unused
step1.log            step3_igprofCPU.1.gz        step3_igprofMEM.399.gz  step4_FastTimerService.log  step4_igprofMEM.200.gz  step5_circles.json          step5_igprofMEM.1.gz    step5_TimeMemoryInfo.log
step1.root.unused    step3_igprofCPU.200.gz      step3_igprof_mem.txt    step4_igprofCPU.1.gz        step4_igprofMEM.399.gz  step5_FastTimerService.log  step5_igprofMEM.200.gz  step6_JobReport.xml
step2_JobReport.xml  step3_igprofCPU.399.gz      step3_JobReport.xml     step4_igprofCPU.200.gz      step4_igprof_mem.txt    step5_igprofCPU.1.gz        step5_igprofMEM.399.gz  step6.log
step2.log            step3_igprofCPU.gz          step3.log               step4_igprofCPU.399.gz      step4_JobReport.xml     step5_igprofCPU.200.gz      step5_igprof_mem.txt    step6.root.unused
step2.root.unused    step3_igprof_cpu.txt        step3.root.unused       step4_igprofCPU.gz          step4.log               step5_igprofCPU.399.gz      step5_JobReport.xml

/eos/cms/store/user/cmsbuild/profiling/data/CMSSW_12_1_0_pre5/slc7_amd64_gcc900/34834.21:
cmdLog_profiling.sh  step2.log                   step3_igprofCPU.50.gz  step3_igprofMEM.50.gz  step3.root.unused           step4_igprofCPU.50.gz  step4_igprofMEM.50.gz  step4.root.unused
step1_JobReport.xml  step2.root.unused           step3_igprofCPU.99.gz  step3_igprofMEM.99.gz  step3_TimeMemoryInfo.log    step4_igprofCPU.99.gz  step4_igprofMEM.99.gz  step4_TimeMemoryInfo.log
step1.log            step3_circles.json          step3_igprofCPU.gz     step3_igprof_mem.txt   step4_circles.json          step4_igprofCPU.gz     step4_igprof_mem.txt   step5_JobReport.xml
step1.root.unused    step3_FastTimerService.log  step3_igprof_cpu.txt   step3_JobReport.xml    step4_FastTimerService.log  step4_igprof_cpu.txt   step4_JobReport.xml    step5.log
step2_JobReport.xml  step3_igprofCPU.1.gz        step3_igprofMEM.1.gz   step3.log              step4_igprofCPU.1.gz        step4_igprofMEM.1.gz   step4.log              step5.root.unused

$ ls /eos/project/c/cmsweb/www/reco-prof/cgi-bin/data/releases/CMSSW_12_1_0_pre5/*/*
/eos/project/c/cmsweb/www/reco-prof/cgi-bin/data/releases/CMSSW_12_1_0_pre5/11834.21/step3:
cpu_endjob.sql3  mem_live.1.sql3  mem_live.200.sql3  mem_live.399.sql3

/eos/project/c/cmsweb/www/reco-prof/cgi-bin/data/releases/CMSSW_12_1_0_pre5/11834.21/step4:
cpu_endjob.sql3  mem_live.1.sql3  mem_live.200.sql3  mem_live.399.sql3

/eos/project/c/cmsweb/www/reco-prof/cgi-bin/data/releases/CMSSW_12_1_0_pre5/11834.21/step5:
cpu_endjob.sql3  mem_live.1.sql3  mem_live.200.sql3  mem_live.399.sql3

/eos/project/c/cmsweb/www/reco-prof/cgi-bin/data/releases/CMSSW_12_1_0_pre5/34834.21/step3:
cpu_endjob.sql3  mem_live.1.sql3  mem_live.50.sql3  mem_live.99.sql3

/eos/project/c/cmsweb/www/reco-prof/cgi-bin/data/releases/CMSSW_12_1_0_pre5/34834.21/step4:
cpu_endjob.sql3  mem_live.1.sql3  mem_live.50.sql3  mem_live.99.sql3
```

@xoqhdgh1002 Please propagate these results to the web page when you have a chance. Thank you!

**Comment by jpata** [comment]
kind ping on this. if you can update already on Tuesday, we can share the results with the Offline/Computing management on Wednesday.

**Comment by xoqhdgh1002** [comment]
I'm sorry I checked your message today. I update PR for 12_1_0_pre5. I hope it's not too late

**Comment by jpata** [comment]
This has been solved in #31, thank you!
